lua/nvim-treesitter/incremental_selection.lua

```diff
@@ -157,9 +157,9 @@ function M.detach(bufnr)
   for f, mapping in pairs(config.keymaps) do
     if mapping then
       if f == "init_selection" then
-        vim.keymap.del("n", mapping, { buffer = bufnr })
+        pcall(vim.keymap.del, "n", mapping, { buffer = bufnr })
       else
-        vim.keymap.del("x", mapping, { buffer = bufnr })
+        pcall(vim.keymap.del, "x", mapping, { buffer = bufnr })
```

Comment:
I'm bit worried that this hides an actual bug. Under which circumstance are you encountering this error?

Reply:
@theHamsta Not the author, but I'm seeing this when using vim-unimpaired's `[l` and `]l` mappings to cycle through the location list.